<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>552</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Gotham Pro</family>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font: 8pt &quot;Gotham Pro&quot;;
	background-color: white;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton538">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{

background:white;
border-radius:20px;
border:2px solid black;
font: 18pt &quot;Gotham Pro&quot;;
}</string>
   </property>
   <property name="text">
    <string>Перевести</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit534">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>511</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Gotham Pro&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_533">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>511</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Gotham Pro&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_536">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Gotham Pro&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label531">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Gotham Pro&quot;;</string>
   </property>
   <property name="text">
    <string> Ответ(кг):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_532">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Gotham Pro&quot;;</string>
   </property>
   <property name="text">
    <string>  Введите плотность (кг/м3)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_533">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Pro</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Gotham Pro&quot;;</string>
   </property>
   <property name="text">
    <string>       Введите объём (м3)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
